base_set/cards/cellar.py

```diff
@@ -0,0 +1,36 @@
+from core.card import CardType
+from core.card import KingdomCard
+from core.counters import CounterId
+from core.counters import CounterName
+from core.decision import Decision
+from core.locations import Location
+from core.locations import LocationName
+
+
+class CellarDecision(Decision):
+    def __init__(self, game_state):
+        player = game_state.get_current_player_name()
+        cards = list(game_state.get_location(Location(player, LocationName.HAND)))
+        self.options = cards
+        self.min = 0
+        self.max = min(len(cards), 4)
```

Comment:
There is no maximum here, guessing you copied the ChapelDecision which says "trash up to 4 cards" :)